suzieq/poller/worker/services/interfaces.py

```diff
@@ -742,6 +743,15 @@ def _clean_iosxr_data(self, processed_data, raw_data):
                 entry['speed'] = NO_SPEED
                 continue
 
+            if not 'type' in entry:
+                # We have some weird outputs when some ephemeral interfaces
+                # such as NDE_0 popup between the commands we run causing us
+                # to not have full data about this interface. Drop such
+                # interfaces. If they're not ephemeral, we'll get them in the
+                # next run
+                drop_indices.append(i)
+                return
```

Comment:
Pylint is complaining here, we should return an expression. We might:
1. return an empty dictionary
2. use `# pylint: disable=inconsistent-return-statements`

Reply:
```suggestion
                continue
```
Shouldn't this return be a `continue`? In this way we discard all the interfaces

Reply:
YES!! I found and fixed it already